htdocs/product/stock/card.php

```diff
@@ -618,19 +628,27 @@
 						print '<td class="center"><a href="'.DOL_URL_ROOT.'/product/stock/product.php?dwid='.$object->id.'&id='.$objp->rowid.'&action=correction&backtopage='.urlencode($_SERVER["PHP_SELF"].'?id='.$id).'">';
 						print $langs->trans("StockCorrection");
 						print "</a></td>";
-					}
-
-					print "</tr>";
-					$i++;
-				}
-				$db->free($resql);
+                        print '<td align="center"><a href="' . DOL_URL_ROOT . '/product/stock/product.php?dwid=' . $object->id . '&id=' . $objp->rowid . '&action=correction&backtopage=' . urlencode($_SERVER["PHP_SELF"] . '?id=' . $id) . '">';
```

Comment:
Still wrong.
The column "print $langs->trans("StockCorrection");" is now duplicated.

Reply:
Fixed